<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ingredients Dialog</string>
  </property>
  <widget class="QTableWidget" name="ingredients_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>671</width>
     <height>271</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Unit of Measure</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="remove_selected_ingredient">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_ingredient">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>290</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ingredient_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter ingredient name here...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ingredient_quantity">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>42</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="ingredient_unit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>290</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>single</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>slice</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>whole</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cup</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>lb</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>oz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>teaspoon</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>tablespoon</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Unit of Measure:</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>360</y>
     <width>193</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
